projects/core/src/checkout/store/checkout-state.ts

```diff
@@ -34,8 +35,14 @@ export interface CheckoutStepsState {
   orderDetails: Order;
 }
 
+export interface PaymentTypesState {
+  entities: { [code: string]: PaymentType };
+  selected: string;
+}
```

Comment:
Is PaymentTypesState's purpose to store the list of possible Payment types the user can chose from?

In this case, isn't the selected payment type stored somewhere else less global, like on the cart.

Reply:
yes, it should be in 'cart'. I will remove this "selected"